app/src/store.tsx

```diff
@@ -63,6 +80,11 @@ const bcReducer = (state: BCState, action: ReducerAction<BCDispatchAction>): BCS
       const developer = { ...state.developer, environment }
       return { ...state, developer }
     }
+    case AddCredentialDispatchAction.ADD_CREDENTIAL_PRESSED: {
```

Comment:
Why does this need to be a dispatch?

Reply:
It was the cleanest way I could find to do it, since the event is triggered in the navigator and I need a component in the credential list screen to react to it